models/response/draft_transaction.go

```diff
@@ -0,0 +1,43 @@
+package response
+
+import (
+	"time"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/common"
+)
+
+const (
+	// DraftStatusDraft is when the transaction is a draft
+	DraftStatusDraft string = "draft"
+
+	// DraftStatusCanceled is when the draft is canceled
+	DraftStatusCanceled string = "canceled"
+
+	// DraftStatusExpired is when the draft has expired
+	DraftStatusExpired string = "expired"
+
+	// DraftStatusComplete is when the draft transaction is complete
+	DraftStatusComplete string = "complete"
+)
+
+// DraftTransaction is a model that represents a draft transaction.
+type DraftTransaction struct {
+	// Model is a common model that contains common fields for all models.
+	common.Model
```

Comment:
Assuming that DraftTransaction struct is the camelCase model, then it should include also new version of common.Model, because there are some snake case fields in it.